### ISSUE TITLE
508/a11y: OES Project Process page - images lacking alt attribute

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#340]
On the [OES Project Process page](https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/projectprocess/), there are six icon images which do not have an `alt` attribute. Those images are adjacent to text for the steps of the process, so they may be considered decorative. In order to properly indicate decorative images, an empty `alt` must be added to each decorative `img` element. 

Relevant standard:
[508/WCAG 1.1.1 Non-text Content (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content)

**Comment by natastro** [comment]
Hello! I'd be happy to help with this if needed. Thank you! 😀

**Comment by iamjolly** [comment]
@natastro - I'm about to commit a fix for these, but thank you for the offer!
